addons/mail/tests/test_mail_render.py

```diff
@@ -330,62 +338,72 @@ def test_template_rendering_various(self):
         # static xml
         src = '<p class="text-muted"><span>This is a string</span></p>'
         expected = '<p class="text-muted"><span>This is a string</span></p>'
-        for engine in ['jinja', 'qweb']:
+        for engine in ['inline_template', 'qweb']:
             result = MailRenderMixin._render_template(
                 src, partner._name, partner.ids, engine=engine,
             )[partner.id]
-            self.assertEqual(expected, result if engine == 'jinja' else result)  # tde: checkme
+            self.assertEqual(expected, result)  # tde: checkme
 
         # code xml
         srces = [
-            '<p class="text-muted"><span>This is a string with a number ${13+13}</span></p>',
+            '<p class="text-muted"><span>This is a string with a number {{ 13+13 }}</span></p>',
             '<p class="text-muted"><span>This is a string with a number <t t-out="13+13"/></span></p>',
         ]
         expected = '<p class="text-muted"><span>This is a string with a number 26</span></p>'
-        for engine, src in zip(['jinja', 'qweb'], srces):
-            result = MailRenderMixin._render_template(
-                src, partner._name, partner.ids, engine=engine,
-            )[partner.id]
-            self.assertEqual(expected, result if engine == 'jinja' else result)  # tde: checkme
-
-        # condition block xml
-        src = """<b>Test</b>
-% if False:
-    <b>Code not executed</b>
-% endif"""
-        expected = "<b>Test</b>\n"  # TDE: to check
-        for engine in ['jinja']:
+        for engine, src in zip(['inline_template', 'qweb'], srces):
             result = MailRenderMixin._render_template(
                 src, partner._name, partner.ids, engine=engine,
             )[partner.id]
-            self.assertEqual(expected, result)
+            self.assertEqual(expected, str(result))
 
-        # complete block xml
```

Comment:
Same: keep and update for QWeb testing

Reply:
This test if we can set a variable. I added a test for qweb but how would I do it in for `inline_template` as we cannot longer do it ?